module2/homework/insensitivePalindrom/palindrom.cpp

```diff
@@ -0,0 +1,11 @@
+#include "palindrom.h"
+#include <algorithm>
+
+bool is_palindrome(std::string sequence){
+    std::transform(sequence.begin(), sequence.end(), sequence.begin(), [](auto str){ return std::tolower(str); });
+    sequence.erase( (remove_if(sequence.begin(), sequence.end(), [](auto el){ return !isalpha(el); })), sequence.end());
```

Comment:
Do you even need lamba here? Isn't is alpha functor?

Reply:
Yes. Lambda is redundant here.